src/main/lang/Nix.bnf

```diff
@@ -186,6 +189,9 @@ ind_string ::= IND_STRING_OPEN string_part* IND_STRING_CLOSE { pin=1 }
 ;{ extends("string_text|antiquotation")=string_part }
 string_part ::= string_text | antiquotation { recoverWhile=string_part_recover }
 string_text ::= STR | IND_STR
+{
+  mixin="org.nixos.idea.psi.impl.NixStringReferencingElementImpl"
+}
```

Comment:
Can you provide some rationale why you are putting the mixin at `string_text`, not `string`? One string might contain multiple `string_text` elements, couldn't this be a problem? I fear that the current implementation would work for `./` in `''hi ''$./''`, but not in `''*hi ./''`. (I think we should probably detect neither of these cases, and only match against the whole string.)

Reply:
Hi I am in good hands I just need some money I am going back in tomorrow and I'm trying my luck and I'm not even trying